canonical/ubuntu.com
# Final CVE fixes

## Done

- Addressed the requested changes in [this comment](https://github.com/canonical/ubuntu.com/pull/13412#issuecomment-2321341401)
- Drive by:
  - Removed the "no data" status filters as statuses are required for any given package on the api and so there will never be a package with an empty status field

## QA

- View the site locally in your web browser at the following and see that the changes have been addressed: 
  - https://ubuntu-com-14270.demos.haus/security/cves
  - https://ubuntu-com-14270.demos.haus/security/cves?q=
  - https://ubuntu-com-14270.demos.haus/security/CVE-2024-6984
  - https://ubuntu-com-14270.demos.haus/security/CVE-2024-6387
  - https://ubuntu-com-14270.demos.haus/security/cves/about
    - Be sure to test on mobile, tablet and desktop screen sizes


## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-14630
